### PR TITLE
IDEX-3928: Add file filter to Git log command

### PR DESCRIPTION
**Pull request by vinokurig**
@vparfonov  @skabashnyuk please review

### CODE REVIEW

File: che-core-git-impl-native/src/main/java/org/eclipse/che/git/impl/nativegit/commands/LogCommand.java [hunk 1]
```diff
@@ -46,6 +47,9 @@ public LogCommand(File place) {
         if (count > 0) {
             commandLine.add("-" + count);
         }
+        if (fileFilter != null && !fileFilter.isEmpty()) {
```
Comment (sleshchenko) [inline che-core-git-impl-native/src/main/java/org/eclipse/che/git/impl/nativegit/commands/LogCommand.java]: Please take a look implementation of CommandLine https://github.com/codenvy/che-core/blob/master/platform-api/che-core-api-core/src/main/java/org/eclipse/che/api/core/util/CommandLine.java#L62-L67
I think we can miss these checkings

Comment (vinokurig) [inline che-core-git-impl-native/src/main/java/org/eclipse/che/git/impl/nativegit/commands/LogCommand.java]: done

**Comment by skabashnyuk** [comment]
ok for me. @vparfonov ?

**Comment by vparfonov** [comment]
ok

**Pull request merged**